specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/examples/apps/SuccessfulPublishApplicationRequest.json

```diff
@@ -13,8 +13,8 @@
       "headers": {},
       "body": {
         "isStaging": false,
-        "endpointUrl": "https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/11be6373fca44ded80fbe2afa8597c18",
-        "assignedEndpointKey": "11be6373fca44ded80fbe2afa8597c18",
+        "endpointUrl": "https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/c555c6ef-974f-4d73-b573-69feeb4a85f8",
+        "assignedEndpointKey": null,
```

Comment:
Should this value really be null or some dummy value?  The point of examples is to illustrate service behavior. If the service would actually return a value here, I would recommend using some kind of obvious dummy value.

Reply:
It actually returns null. This is a legacy property that hasn't been populated for a while.